packages/gallery/src/components/item/itemView.js

```diff
@@ -22,6 +22,9 @@ import {
   getImageStyle,
 } from './itemViewStyleProvider';
 import VideoItemWrapper from './videos/videoItemWrapper';
+import withMagnified from '../hoc/withMagnified.js';
+
+const MagnifiedImageItem = withMagnified(ImageItem);
```

Comment:
@IzaacAyelin 
Why does the `itemView` knows the magnifier? Didn't we say that because `itemView` can be a video or text - the magnifier will be inside of `imageItem`?

Reply:
we have some other places that the ImageItem that I didn't notice at first, so I couldn't export it directly with the hoc